spring-kafka/src/test/java/org/springframework/kafka/kstream/KafkaStreamsTests.java

```diff
@@ -80,8 +80,8 @@
 	@Autowired
 	private SettableListenableFuture<String> resultFuture;
 
-	@Autowired
-	private KafkaEmbedded kafkaEmbedded;
+//	@Autowired
+//	private KafkaEmbedded kafkaEmbedded;
```

Comment:
Ha! Good point!
Now I recall that I did that to add a sample to the docs.
I'll polishing it during merge.
Thank you!